Formula/duc.rb

```diff
@@ -0,0 +1,39 @@
+class Duc < Formula
+  desc "Suite of tools for inspecting disk usage"
+  homepage "http://duc.zevv.nl"
+  url "https://github.com/zevv/duc/releases/download/1.4.3/duc-1.4.3.tar.gz"
+  sha256 "504810a1ac1939fb1a70bd25e492f91ea38bcd58ae0a962ce5d35559d7775e74"
+  head "https://github.com/zevv/duc.git"
+
+  option "with-opengl", "Enable OpenGL gui"
```

Comment:
We don't want to provide 2 different GUIs, so either OpenGL is compatible with cairo/pango and we enable it all the time, or it's incompatible and we drop it.